packages/fx-core/src/core/middleware/envInfoWriter.ts

```diff
@@ -20,7 +19,7 @@ export function EnvInfoWriterMW(skip = false): Middleware {
     try {
       await next();
     } catch (e) {
-      if ((e as any)["name"] === getStrings().solution.CancelProvision) throw e;
+      if ((e as any)["name"] === "CancelProvision") throw e;
```

Comment:
Is it better to extract the constant (or to an error class) into `solution/fx-solution/constants.ts` instead of hard coding in different places?

Reply:
Sure. There exists the constant, I will improve later.